power-pay-frontend/src/components/pinInput.tsx

```diff
@@ -0,0 +1,33 @@
+const PinInput: React.FC = () => {
+  
+
+
+ 
+  return (
+    <div className="flex justify-center items-center  mb-34  bg-800 text-black text-sm">
+      <div className="">
+        <div className="flex rounded-lg w-80 m-auto px-4 py-2  text-lg absolute  inset-x-0 top-4 bg-gray-100">
+          <div>
+            <h1 className="text-lg text-center text-red-400 pb-5">Notice!!!</h1>
+            <p className="text-sm text-center text-black-400 pb-4">By entering you PIN below, you confirm that the sum of of FCFA100.50  be sent to Mr Stephane</p>
```

Comment:
hello @Motouom please why is the success message hard coded in ```By entering you PIN below, you confirm that the sum of of FCFA100.50 be sent to Mr Stephane```

Reply:
Its just a text to fill the blank while waiting for what is actually going to be implemented there @AssahBismarkabah